src/solver/optimisation/opt_gestion_second_membre_cas_lineaire.cpp

```diff
@@ -61,19 +61,14 @@ static void shortTermStorageCumulationRHS(
                 for (const auto& constraint: additional_constraints.constraints)
                 {
                     const int cnt = CorrespondancesDesContraintesHebdomadaires
-                            .ShortTermStorageCumulation[constraint.globalIndex];
-
-                    SecondMembre[cnt] = std::accumulate(constraint.hours.begin(),
-                                                        constraint.hours.end(),
-                                                        0.0,
-                                                        [weekFirstHour, &additional_constraints
-                                                        ](
-                                                        const double sum,
-                                                        const int hour)
-                                                        {
-                                                            return sum + additional_constraints.rhs[
-                                                                    weekFirstHour + hour - 1];
-                                                        });
+                                      .ShortTermStorageCumulation[constraint.globalIndex];
+
+                    SecondMembre[cnt] = std::accumulate(
+                      constraint.hours.begin(),
+                      constraint.hours.end(),
+                      0.0,
+                      [weekFirstHour, &additional_constraints](const double sum, const int hour)
```

Comment:
here you should check if the constraint is enabled

Reply:
There is no need for this, because the addition of constraints to `shortTermStorageInput` is already done in a selective manner (see sim_calcul_economique.cpp)